rados/conn.go

```diff
@@ -91,6 +91,19 @@ func (c *Conn) OpenIOContext(pool string) (*IOContext, error) {
 	}
 }
 
+// Check if a pool exists
+func (c *Conn) PoolExists(poolName string) bool {
+    c_pool := C.CString(poolName)
+    defer C.free(unsafe.Pointer(c_pool))
+    ret := int(C.rados_pool_lookup(c.cluster, c_pool))
```

Comment:
Cast it to int64_t 
CEPH_RADOS_API int64_t rados_pool_lookup(rados_t cluster, const char * pool_name)